jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/inputs/SelectAssert.java

```diff
@@ -1,52 +1,9 @@
 package com.epam.jdi.light.material.asserts.inputs;
 
-import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
 
 import com.epam.jdi.light.asserts.generic.UIAssert;
-import com.epam.jdi.light.common.JDIAction;
 import com.epam.jdi.light.material.elements.inputs.Select;
-import org.hamcrest.Matchers;
-
-import java.util.List;
 
 public class SelectAssert extends UIAssert<SelectAssert, Select> {
```

Comment:
Do you really need this class?

Reply:
yes, it used in Select class as extends class